src/main/scripts/org/reaktivity/specification/mqtt/reject.connect.with.multiple.topic.alias.maximum/server.rpt

```diff
@@ -0,0 +1,38 @@
+#
```

Comment:
Rename to match `reject...when...` format.
Suggest `reject.connect.when.topic.alias.maximum.repeated`